tests/test_datasets/test_dataset_functions.py

```diff
@@ -1162,6 +1162,8 @@ def test___publish_fetch_ignore_attribute(self):
         self.__class__.test_publish_fetch_ignore_attribute_did = upload_did
         self.__class__.test_publish_fetch_ignore_attribute_list = ignore_attribute
 
+    # with concurrent runs this function maybe called before test__publish_fetch_ignore_attribute
+    @pytest.mark.flaky(reruns=5)
```

Comment:
You could instead add a wait loop here to make sure this one doesn't fail due to a dataset not yet processed (like for example this one: https://github.com/openml/openml-python/blob/develop/tests/test_runs/test_run_functions.py#L67)

Reply:
Is it okay if in the current while loop implementation I add a `time.sleep(10)` before the `continue`?